ArduCopter/system.cpp

```diff
@@ -223,7 +223,7 @@ void Copter::init_ardupilot()
     // disable safety if requested
     BoardConfig.init_safety();
 
-    hal.console->printf("\nReady to FLY ");
```

Comment:
i think we should make this a send_text with "ArduPilot Ready" in libraries/AP_Vehicle and remove the printf from all vehicles